### PR TITLE
use Blob instead of deprecated BlobBuilder in lib/client/platform/cordova/2.0.0/bridge/file.js

### PR DESCRIPTION
**Pull request by maxme**
BlobBuilder is now deprecated, use new Blob constructor instead

### CODE REVIEW

**Comment by dansilivestru** [comment]
thanks for the contribution :-) I'll review and test it today. Just looking into whether or not we'll need you to sign a CLA before we can pull this in. In the mean time, details can be found here: http://blackberry.github.com/howToContribute.html

I'll keep you posted

File: lib/client/platform/cordova/2.0.0/bridge/file.js [hunk 1]
```diff
@@ -240,15 +240,14 @@ module.exports = {
...
-            bb = new BB();
+            blob;
 
         // Format source path
         sourcepath = ((file.fullPath ? file.fullPath : '') + file.name);
         sourcepath = cleanPath(sourcepath);
 
         // Create a blob for the text to be written
-        bb.append(text);
+        blob = new Blob([text]);
```
Comment (dansilivestru) [inline lib/client/platform/cordova/2.0.0/bridge/file.js:250]: We should probably update this to:

```
 blob = new Blob([text], {type: "text/plain"})
```

to be safe

**Comment by dansilivestru** [comment]
Hi @maxme, Thanks for your contribution.

The Individual Contribution Agreement (and the CCLA for corporations) are our preferred way to incorporate contributions from the community. We track all contributors in the list of signatories page, which also gives you an opportunity to share your twitter handle and website with the community. Additionally many people share their approximate location and we create this geomap.

That said, section 5 of the ASL2 covers intentional submissions into the code base and small contributions like yours can be accepted under that clause.

To sign the ICLA, print it, fill it in, scan it into a PDF and then mail it to us. In the meantime and regardless, we will merge your contribution under section 5.

I will merge this in now (with the small addition I proposed in the comments).

**Comment by dansilivestru** [comment]
Closing as I am merging this manually into the next branch first.

**Comment by pelegri** [comment]
I'm recording this as an actual contribution for the purposes of the Dev Alpha C device points.

**Pull request closed without merging**